examples/fast-api/fast_api.py

```
@@ -0,0 +1,138 @@
+            async with httpx.AsyncClient() as client:
+                await client.post(self.logs_url, json=log_entry, timeout=5.0)
+        except Exception:
+            pass  # Don't break app if logging fails
+
+
+# Initialize logger
+logger = AlloyLogger("http://grafana-alloy-service.railway.internal:3100")
+
+
+# Metrics endpoint for Alloy to scrape (optional)
+@app.get("/metrics")
```

Ruff (0.11.9)

Undefined name `app`

(F821)